src/modules/rabbitmq/elastic-updates/scam-trigger/scam-update-events.consumer.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { rabbitExchanges, rabbitQueues } from '../../rabbit-config';
+import { MarkScamCollectionEvent } from './markScamCollection.event';
+import { CompetingRabbitConsumer } from '../../rabbitmq.consumers';
+import { CollectionScamService } from 'src/modules/scam/collection-scam.service';
+import { ScamInputEnum } from 'src/modules/admins/models/scam-update.input';
+@Injectable()
+export class ScamUpdateEventsConsumer {
+  constructor(private readonly collectionScamService: CollectionScamService) { }
+
+  @CompetingRabbitConsumer({
+    connection: 'common',
+    queueName: rabbitQueues.SCAM_UPDATE,
+    exchange: rabbitExchanges.SCAM_UPDATE,
+  })
+  async consumeScamEvents(event: MarkScamCollectionEvent) {
+    if (event.type === ScamInputEnum.allow) {
+      await this.collectionScamService.manuallyClearCollectionScamInfo(event.collectionIdentifier)
+    } else {
```

Comment:
Could have also been "else if" for "deny", and throw error (unrecognized) on "else".

Reply:
I would like to let it like this because are only two possibilities and in the future if we add another one then update the condition :pray: